frontend/src/components/Form/index.jsx

```diff
@@ -0,0 +1,636 @@
+import React, { useState } from 'react'
+import FormField from './FormField'
+import Button from '../Button'
+import './styles.css'
+
+const Form = () => {
+  const [formData, setFormData] = useState({
```

Comment:
Tinha algum outro jeito de gerenciar as mudanças de estado no React sem usar o objeto inteiro?
Dúvida genuína @VladeMelo , n to reclamando nem nada